ACMILabs/acmi-api
# #92 Fix works tsv creators ids

Resolves #90

To make it easier to deduplicate Creators with external sources like Wikidata, let's generate a `creators.tsv` file.

### Acceptance Criteria
- [x] In the works tsv for `creators_primary` and `creators_other` use the `creator_id` rather than the `role_in_work` id

### Testing instructions
1. Start the base container: `make base`
1. Generate a `works.tsv` (see below or use the one I generated)
1. Open the tsv: `open app/tsv/works.tsv`
1. Not the `creators_primary` now show Creator IDs: e.g. [23955](https://xos.acmi.net.au/api/creators/23955/)

```python
$ docker exec -it api python

from app.api import XOSAPI
xos_private_api = XOSAPI()
xos_private_api.generate_tsv('works')
```
